readme.md

```diff
@@ -60,6 +60,43 @@ cd ../lighthouse-cli/
 npm link lighthouse-core
 ```
 
+## Custom run configuration
+
+You can supply your own run configuration to customize what audits you want details on. Copy the [default.json](https://github.com/GoogleChrome/lighthouse/blob/master/lighthouse-core/config/default.json) and start customizing. Then provide to the CLI with `lighthouse --config-path=$PWD/myconfig.json <url>`
+
+## Trace processing
+
+Lighthouse can be used to analyze trace and performance data collected from other tools (like WebPageTest and ChromeDriver). The `traceContents` and `performanceLog` artifact items can be provided using a string for the absolute path on disk. The perf log is captured from the Network domain (a la ChromeDriver's [`enableNetwork` option](https://sites.google.com/a/chromium.org/chromedriver/capabilities#TOC-perfLoggingPrefs-object) and reformatted slightly. As an example, here's a trace-only run that's also evaluating just user-timings:
+
+##### `config.json`
+```js
+{
+ "audits": [
+  "user-timings"
+ ],
+
+ "artifacts": {
+   "traceContents": "~/code/lighthouse-core/test/fixtures/traces/trace-user-timings.json",
+   "performanceLog": "~/code/lighthouse-core/test/fixtures/traces/perflog.json"
+ },
+
+ "aggregations": [{
+   "name": "Performance Metrics",
+   "description": "These encapsulate your app's performance.",
+   "scored": false,
+   "categorizable": false,
+   "items": [{
+     "criteria": {
+       "user-timings": { "rawValue": 0, "weight": 1 }
```

Comment:
You can add critical requests paths here as well to show what the performance logs can be used for. This would include adding it to audits, and as an additional criteria under the Performance Metrics aggregations